mirdata/datasets/ballroom.py

```diff
@@ -164,6 +168,20 @@ def to_jams(self):
         )
 
 
+def get_genre(audio_path: Optional[str]) -> Optional[str]:
```

Comment:
normally, for consistency, we name this functions `load_<thing>()`. I think it would be nice if you could use this convention here as well